src/operator/numpy/np_matrix_op.cc

```diff
@@ -1081,5 +1158,121 @@ NNVM_REGISTER_OP(_npi_hsplit_backward)
 })
 .set_attr<FCompute>("FCompute<cpu>", HSplitOpBackward<cpu>);
 
+bool NumpyAppendType(const nnvm::NodeAttrs& attrs,
+                     std::vector<int> *in_type,
+                     std::vector<int> *out_type) {
+  const int num_args = 2;
+  CHECK_EQ(in_type->size(), num_args);
+  CHECK_EQ(out_type->size(), 1);
+  int dtype = -1;
+  for (int i = 0; i < num_args; i++) {
+    if (dtype == -1) {
+      dtype = in_type->at(i);
+    }
+  }
+  if (dtype == -1) {
+    dtype = out_type->at(0);
+  }
+  for (int i = 0; i < num_args; i++) {
+    TYPE_ASSIGN_CHECK(*in_type, i, dtype);
+  }
+  TYPE_ASSIGN_CHECK(*out_type, 0, dtype);
+  return dtype != -1;
+}
+
+bool NumpyAppendShape(const nnvm::NodeAttrs& attrs,
+                 mxnet::ShapeVector *in_shape,
+                 mxnet::ShapeVector *out_shape) {
```

Comment:
Alignment:
```c++
bool NumpyAppendShape(const nnvm::NodeAttrs& attrs,
                      mxnet::ShapeVector *in_shape,
                      mxnet::ShapeVector *out_shape) {
```

Reply:
Also check all other functions for alignement

Reply:
> Also check all other functions for alignment

ok